PSReadLine/Render.cs

```diff
@@ -974,6 +982,11 @@ private string GetTokenColor(Token token)
                 return _options._memberColor;
             }
 
+            if ((token.Kind == TokenKind.Identifier | token.Kind == TokenKind.Generic) && token.TokenFlags == TokenFlags.None)
+            {
+                return _options._commandArgumentColor;
+            }
```

Comment:
I don't think so, I've pasted a lot of code into the console to test the colors and I haven't found any unexpected results regarding command arguments missing color or affecting something that wasn't a command argument.

Reply:
I would suggest using logical or instead of bitwise or.

That said, with these changes, which tokens get the default now? I can't think of any, in which case, it seems unnecessary to introduce a new option.